engines/bops_admin/app/views/bops_admin/tokens/_file_downloader.html.erb

```diff
@@ -0,0 +1,20 @@
+<% if file_downloader.errors.any? %>
+  <a name="token-file-downloader-field-error"></a>
+<% end %>
```

Comment:
The automatic link generation for error messages doesn't seem to work for StoredModel nested fields and didn't want to go down a rabbit hole.